src/Core/Grid/Column/Type/Common/EnabledOrNotColumn.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Column\Type\Common;
+
+use PrestaShop\PrestaShop\Core\Grid\Column\AbstractColumn;
+
+/**
+ * Column used to display booleans as icons for "enabled" or "active" properties
+ */
+final class EnabledOrNotColumn extends AbstractColumn
```

Comment:
hmm, maybe we can name it `StatusColumn` or similar? :/

Reply:
In fact I realized it should be `ToggleColumn` because in the legacy page, this column allows to toggle the value of the boolean property `active`being displayed